test/functional/module_stream/list_test.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/module_stream'
+
+module HammerCLIKatello
+  describe ModuleStreamCommand::ListCommand do
+    it 'allows minimal options' do
```

Comment:
```rb
it 'resolves repository ID from repo name and product ID'
it 'resolves repository ID from repo name, product name, and org ID'
it 'supports --search'
it 'allows --content-view-version-id'
```

Reply:
+1 to all of the above except search isn't implemented yet

Reply:
If search isn't implemented, and you aren't planning to implement it, we should remove that option from the subcommand.

Reply:
yeah, I'm fine with it. I really want to avoid releasing a version of hammer with these 'broken' options, so as long as all those other PRs make it into the release, we're fine.